### PR TITLE
fix: issue 20810 Line 1 is written in the tooltip of the edit button in Bulk Publish

### PR DESCRIPTION
**Pull request by archanaagivale30**
### What does it do?

Fix: [Design] Line 1, line 2, line 3… is written in the tooltip of the edit button in Bulk Publish

### Why is it needed?

Expected: Only “Edit item” should appear in the tooltip

### How to test it?

- Go to Content Type 
- Select multiple records
- Click on publish button
- Over on edit button in modal

### Related issue(s)/PR(s)

#20810

### CODE REVIEW

**Comment by lucasboilly** [comment]
Hello @archanaagivale30 and @simotae14 :)

I think we should make it even simpler and write "Edit". Apart from that, that's great! Thanks!

**Comment by archanaagivale30** [comment]
@simotae14 @lucasboilly Updated suggested changes .

**Comment by lucasboilly** [comment]
Hey @archanaagivale30, it's perfect! Thanks again :) cc @simotae14

File: packages/core/content-manager/admin/src/pages/ListView/components/BulkActions/PublishAction.tsx [hunk 1]
```diff
@@ -248,18 +248,10 @@ const SelectedEntriesTableContent = ({
...
-                        {
-                          id: 'content-manager.components.ListViewHelperPluginTable.row-line',
-                          defaultMessage: 'item line {number}',
-                        },
-                        { number: index + 1 }
-                      ),
-                    }
-                  )}
+                  label={formatMessage({
+                    id: 'app.component.HelperPluginTable.edit',
```
Comment (simotae14) [inline packages/core/content-manager/admin/src/pages/ListView/components/BulkActions/PublishAction.tsx:252]: Can I ask you to add the id also in the translation file packages/core/content-manager/admin/src/translations/en.json ? 🙏🏽

**Review by simotae14 — CHANGES_REQUESTED** [review]
Hey @archanaagivale30 thanks again for your great contribution 🏅 

I asked in the comment just to add the id and value of the message in the translations file to let contributors add the correct content for their language. Thanks again

File: packages/core/content-manager/admin/src/pages/ListView/components/BulkActions/PublishAction.tsx (hunk 1 above)
Comment (archanaagivale30) [reply packages/core/content-manager/admin/src/pages/ListView/components/BulkActions/PublishAction.tsx]: @simotae14 Rather adding new id i used the old one for Edit msg

**Comment by archanaagivale30** [comment]
@simotae14 I used an old ID because the previously mentioned ID is not present in the translation files. So rather than creating a new one I used the already presented ID

File: packages/core/content-manager/admin/src/pages/ListView/components/BulkActions/PublishAction.tsx [hunk 2]
```diff
@@ -248,18 +248,10 @@ const SelectedEntriesTableContent = ({
...
-                      target: formatMessage(
-                        {
-                          id: 'content-manager.components.ListViewHelperPluginTable.row-line',
-                          defaultMessage: 'item line {number}',
-                        },
-                        { number: index + 1 }
-                      ),
-                    }
-                  )}
+                  label={formatMessage({
```
Comment (simotae14) [inline packages/core/content-manager/admin/src/pages/ListView/components/BulkActions/PublishAction.tsx:251]: I suggest you to use something like
`label={formatMessage({
    id: 'content-manager.bulk-publish.edit',
    defaultMessage: 'Edit item',
 })}`
 
 and add the new id here https://github.com/strapi/strapi/blob/develop/packages/core/admin/admin/src/translations/en.json

**Review by simotae14 — CHANGES_REQUESTED** [review]
please have a look to my comment, thanks

**Comment by archanaagivale30** [comment]
@simotae14 added changes as per your suggestion please check.

**Comment by simotae14** [comment]
Hello @archanaagivale30 , thanks again, I need just one little modification, I prefer if you use `Edit item` instead of just `Edit` so can you just change the defaultMessage and the value in the translation file? thanks in advance

**Comment by archanaagivale30** [comment]
> Hello @archanaagivale30 and @simotae14 :)
> 
> I think we should make it even simpler and write "Edit". Apart from that, that's great! Thanks!

@simotae14 the `Edit` text is suggested by @lucasboilly    

https://github.com/strapi/strapi/pull/21715#issuecomment-2446310233

**Comment by simotae14** [comment]
I'm sorry, I missed that part of the conversation. Can I ask you to check locally if the e2e test is failing?

**Comment by archanaagivale30** [comment]
@simotae14 I already checked the e2e test locally but it failed in a different file

**Pull request merged**